mmengine/model/base_module.py

```diff
@@ -176,10 +167,9 @@ def _dump_init_info(self):
                 with_file_handler = True
         if not with_file_handler:
             for name, param in self.named_parameters():
-                print_log(
+                logger.info(
```

Comment:
Why do we replace `print_log` with `logger.info`?

Reply:
Maybe here we must get the logger to use the handler to write messages to files. Since we've got logger, there is not need to use `print_log` anymore.